packages/suite/src/middlewares/suite/suiteMiddleware.ts

```diff
@@ -64,6 +64,11 @@ const suite = (api: MiddlewareAPI<Dispatch, AppState>) => (next: Dispatch) => as
             api.dispatch(routerActions.init());
             // backend connected, suite is ready to use
             api.dispatch(suiteActions.onSuiteReady());
+            // desktopApi ready
```

Comment:
since this is optional only for desktop build, lets place it in some `@suite-desktop/src/support` file.
reason described above (react-native case)